private/bufpkg/bufcheck/bufbreaking/internal/bufbreakingcheck/util.go

```diff
@@ -107,6 +108,80 @@ func newCorpus(
 	}
 }
 
+type cardinality int
+
+const (
+	cardinalityUnknown cardinality = iota
```

Comment:
Global consts always go at top, always start with iota + 1, would put in separate file

Reply:
> always start with iota + 1

So I can skip the "unknown" value then?

> would put in separate file

Just this type? (Like each type in its own file?) Or both of these new enum-like types in the same file?

Reply:
Moved constants to top of file. If you want these types in a separate file (or files), let me know.